ruby/tests/generated_code.rb

```diff
@@ -38,6 +38,7 @@
     repeated :map_string_int32, :message, 69, "A.B.C.TestMessage.MapStringInt32Entry"
     repeated :map_string_bool, :message, 70, "A.B.C.TestMessage.MapStringBoolEntry"
     optional :nested_message, :message, 80, "A.B.C.TestMessage.NestedMessage"
+    unless RUBY_PLATFORM == "java"
```

Comment:
We shouldn't be modifying the checked-in generated-code -- actually this causes a failure of the Ruby code generator test (it's a golden file for comparison). Perhaps, as part of the temporary change, make a copy (generated_code_java.rb) and `require` one or the other from the test based on the platform?